pkg/rbac/userData.go

```diff
@@ -37,6 +35,13 @@ type UserData struct {
 	authzClient v1.AuthorizationV1Interface
 }
 
+// Stuct to keep a copy of users access
+type UserResourceAccess struct {
```

Comment:
Would it be simpler to rename these structs as `UserData` and `UserDataCache`?

Reply:
Done